Tests/Marketplace/zip_packs.py

```diff
@@ -114,61 +114,110 @@ def remove_test_playbooks_from_signatures(path, filenames):
 
-            if gcp_path == BUILD_GCP_PATH:
-                pack_prefix = os.path.join(gcp_path, branch_name, circle_build, 'content', 'packs', pack.name)
-            else:
-                pack_prefix = os.path.join(gcp_path, branch_name, circle_build, pack.name)
-
-            if not branch_name or not circle_build:
-                pack_prefix = pack_prefix.replace('/builds/content', '')
-
-            # Search for the pack in the bucket
-            blobs = list(storage_bucket.list_blobs(prefix=pack_prefix))
-            if blobs:
-                blob = get_latest_pack_zip_from_blob(pack.name, blobs)
-                if not blob:
-                    logging.warning(f'Failed to get the zip of the pack {pack.name} from GCP')
-                    continue
-                download_path = os.path.join(destination_path, f"{pack.name}.zip")
-                zipped_packs.append({pack.name: download_path})
-                logging.info(f'Downloading pack from GCP: {pack.name}')
-                executor_submit(executor, download_path, blob)
-                sleep(1)
-                if os.path.exists('/home/runner/work/content-private/content-private/content/artifacts/'):
-                    logging.info(f"Copying pack from {download_path} to /home/runner/work/content-private/"
-                                 f"content-private/content/artifacts/packs/{pack.name}.zip")
-                    shutil.copy(download_path,
-                                f'/home/runner/work/content-private/content-private/content/artifacts/'
-                                f'packs/{pack.name}.zip')
-            else:
-                logging.warning(f'Did not find a pack to download with the prefix: {pack_prefix}')
+    if gcp_path == BUILD_GCP_PATH:
+        zip_path = os.path.join(zip_path, 'packs')  # directory of the packs
+    elif circle_build:
+        zip_path = os.path.join(zip_path, circle_build)  # directory of the packs
+
+    dir_entries = os.listdir(zip_path)
+    print(f'zip path is: {zip_path}')
+    print(f"List of packs in repo is {', '.join(dir_entries)}")
+    packs_list = [pack.name for pack in os.scandir(PACKS_FULL_PATH)]  # list of all packs from repo
+
+    for entry in dir_entries:
+        entry_path = os.path.join(zip_path, entry)
+        if entry not in IGNORED_FILES and entry in packs_list and os.path.isdir(entry_path):
+            # This is a pack directory, should keep only most recent release zip
+            pack_files = get_pack_files(entry_path)
+            latest_zip = get_latest_pack_zip_from_blob(entry, pack_files)
+            if not latest_zip:
+                logging.warning(f'Failed to get the zip of the pack {entry} from GCP')
+                continue
+            print(f"Found latest zip of {entry}, which is {latest_zip}")
+            zipped_packs.append({Path(latest_zip).stem: latest_zip})
 
     if not zipped_packs:
         logging.critical('Did not find any pack to download from GCP.')
         sys.exit(1)
```

Comment:
Same as above.

Reply:
I guess you're referring to the try-except I used in another function? 
If yes, then I can simply remove the logging message and keep the sys.exit, it will still raise an exception that will be caught by the main function. WDYT?

Reply:
Note the `except Exception` will not catch `SystemExit`.
Reference: https://docs.python.org/3/library/exceptions.html#base-classes

Reply:
Fixed.